app/js/moduleUpload/uploadModule.html

```diff
@@ -319,9 +330,9 @@ <h1 class="output center-block" id="dragAndDropOutput" style="margin-top: -150px
 
         var ext = file.split(".");
         ext = ext[ext.length-1].toLowerCase();
-        var arrayExtensions = ["omod"];
+        var arrayExtensions = ["omod", "zip"];
 
-        if (arrayExtensions.lastIndexOf(ext) == -1) {
+        if (arrayExtensions[0] !== ext && arrayExtensions[1] !== ext) {
```

Comment:
Awesome job @andela-jomadoye. Assuming more file extensions were to be added to the array, checking each one at a time (`arrayExtensions[0] !== ext`) could become painful; so you could use `array.includes` to check if it matches the values already in `arrayExtensions`. Something like:
```
(arrayExtensions.includes(ext)) ? true : false
```
What do you think?